src/components/FileList/FileList.js

```diff
@@ -20,17 +31,189 @@ const FileList = ({fileList, studyId}) => {
+    } else {
+      sortedList =
+        sortMethod === 'createDate'
+          ? createDateSort(fileList, sortDirection)
+          : modifiedDateSort(fileList, sortDirection);
+      if (typeFilterStatus !== '') {
+        sortedList = sortedList.filter(
+          obj => obj.node.fileType === typeFilterStatus,
+        );
+      }
+      if (approvalFilterStatus !== '') {
+        sortedList = sortedList.filter(
+          obj => fileLatestStatus(obj.node) === approvalFilterStatus,
+        );
+      }
+    }
+    if (searchString !== '') {
+      sortedList = sortedList.filter(obj =>
+        obj.node[searchOption]
+          .toLowerCase()
+          .includes(searchString.toLowerCase()),
+      );
+    }
+    return sortedList;
+  };
+
+  const pageCount = Math.ceil(sortedFileList().length / perPage);
+  const pageItems = sortedFileList().slice(
     perPage * (page - 1),
     perPage * (page - 1) + perPage,
   );
 
   return (
     <Fragment>
+      <Segment
+        className="noMargin noHorizontalPadding"
+        basic
+        compact
+        floated="left"
+      >
+        <span className="smallLabel">Filter by:</span>
+        <Dropdown
+          selection
+          clearable
+          selectOnBlur={false}
+          value={approvalFilterStatus}
+          options={statusOptions}
+          placeholder="Choose an approval status"
+          onChange={(e, {value}) => {
+            setApprovalFilterStatus(value);
+          }}
+        />
+      </Segment>
+      <Segment
+        className="noMargin noHorizontalPadding"
+        basic
+        compact
+        floated="left"
+      >
+        <Dropdown
+          selection
+          clearable
+          selectOnBlur={false}
+          value={typeFilterStatus}
+          options={typeOptions}
+          placeholder="Choose a file type"
+          onChange={(e, {value}) => {
+            setTypeFilterStatus(value);
+          }}
+        />
+      </Segment>
+      <Segment
+        className="noMargin noHorizontalPadding"
+        basic
+        compact
+        floated="left"
+      >
+        <span className="smallLabel">Sorted by:</span>
+        <Dropdown
+          selection
+          clearable
+          selectOnBlur={false}
+          value={sortMethod}
+          options={sortOptions}
+          placeholder="Choose an option"
+          onChange={(e, {value}) => {
+            setSortMethod(value);
+          }}
+        />
+        <Button
+          icon
+          basic
+          onClick={() => {
+            if (sortDirection === 'ascending') {
+              setSortDirection('descending');
+            } else {
+              setSortDirection('ascending');
+            }
+          }}
+        >
+          <Icon name={'sort content ' + sortDirection} />
+        </Button>
+      </Segment>
+      <Segment
+        className="noMargin noHorizontalPadding"
+        basic
+        compact
+        floated="right"
+      >
+        <Input
```

Comment:
Out of personal preference, and given that we don't have a huge number of files and descriptions, I think we could instead filter on both fileName and description by concatinating both of them and simply returning any that have the search term in them.
That way, we don't need to be bothered with selecting which field we want to search on.